packages/common/container-definitions/src/audience.ts

```diff
@@ -29,19 +45,9 @@ export interface IAudienceOwner extends IAudience {
  * @remarks Access to the Audience when a container is disconnected is a tricky subject.
  * See the remarks on specific methods for more details.
  *
- * See {@link https://nodejs.org/api/events.html#class-eventemitter | here} for an overview of the `EventEmitter`
- * class.
  * @public
  */
-export interface IAudience extends EventEmitter {
-	/**
-	 * See {@link https://nodejs.dev/learn/the-nodejs-event-emitter | here} for an overview of `EventEmitter.on`.
-	 */
-	on(
-		event: "addMember" | "removeMember",
-		listener: (clientId: string, client: IClient) => void,
-	): this;
-
+export interface IAudience extends IEventProvider<IAudienceEvents> {
```

Comment:
This is breaking, right? I'd expect low chance that someone was using EventEmitter's members on an IAudience object (although `audience.addListener()` instead of `audience.on()` doesn't seem that crazy), but this feels like it needs a changeset.

Additionally, EventeEmitter was the only use of @types/events_pkg in container-definitions so I think we can get rid of that dep. @ChumpChief might have comments or at least be happy about it 😄.

Reply:
adding changeset